ncrypt/crypt_gpgme.c

```diff
@@ -2441,30 +2646,95 @@ int pgp_gpgme_check_traditional(FILE *fp, struct Body *b, bool just_one)
  */
 void pgp_gpgme_invoke_import(const char *fname)
 {
-  gpgme_data_t keydata;
-  FILE *out = NULL;
+  gpgme_ctx_t ctx = create_gpgme_context(false);
+  gpgme_data_t keydata = NULL;
+  gpgme_import_result_t impres;
+  gpgme_import_status_t st;
+  int any;
 
   FILE *in = mutt_file_fopen(fname, "r");
   if (!in)
-    return;
+  {
```

Comment:
The original had `mutt_perror(fname);` here.

Reply:
Thanks, fixed.